psi4/src/psi4/libdpd/dpd.h

```diff
@@ -344,6 +344,7 @@ class PSI_API DPD {
     int trace42_13(dpdbuf4 *A, dpdfile2 *B, int transb, double alpha, double beta);
 
     int file2_init(dpdfile2 *File, int filenum, int irrep, int pnum, int qnum, const char *label);
+    int file2_init(dpdfile2 *File, int filenum, int irrep, int pnum, int qnum, const std::string& label);
```

Comment:
There shouldn't be the need for both of these `file2_init` since a `char*` can be converted to a `std::string`.